src/context/Theme.tsx

```diff
@@ -136,7 +136,7 @@ const darkMode: PaletteOptions = {
 export const zIndexIncrement = 5;
 
 // Need to make the sticky header be on top
-export const stickyHeaderIndex = zIndexIncrement * 20;
+export const stickyHeaderIndex = zIndexIncrement * 100;
```

Comment:
cranking this up to 100 even though it just needed to be above the line below. However, I feel like if we stick with increments of 5/10 we'll be good for a long time until we need to worry about anything being above 100.